js/everything.js

```diff
@@ -147,7 +148,7 @@ var VOLUME_LOCKED = false;
 function onPlayerReady(event) {
     // Only play the video if it's actually there
     if (getCurrentVideoID()) {
-        $("#zen-video-error").text("");
+        hideErrorMessage();
```

Comment:
The error message will flash then disappear wen failing to get the video description here.
Maybe we should set a global var on line 233 for the description error, then only do `$("#zen-video-error").text("")` if that var is unset. My goal here is to persist errors.

What do you think?